doc/sphinx/practical-tools/utilities.rst

```diff
@@ -141,13 +141,19 @@ Including the generated makefile with an include directive is
 discouraged, since the contents of this file, including variable names and
 status of rules, may change in the future.
 
-An optional file ``CoqMakefile.local`` can be provided by the user in order to
-extend ``CoqMakefile``. In particular one can declare custom actions to be
-performed before or after the build process. Similarly one can customize the
-install target or even provide new targets. Extension points are documented in
-paragraph :ref:`coqmakefilelocal`.
-
-The extensions of the files listed in ``_CoqProject`` is used in order to
+Use the optional file ``CoqMakefile.local`` to extend
+``CoqMakefile``. In particular, you can declare custom actions to be
+performed before or after the build process. Similarly you can
```

Comment:
```suggestion
``CoqMakefile``. In particular, you can declare custom actions to
perform before or after the build process. Similarly you can
```

Reply:
Nope, this one changes the meaning in a way that sounds wrong.  The make program performs the actions declared.  The actions don't perform anything.  Replacing "be performed" with "run" would work, if you think that's better.

Reply:
"run" may be better

Reply:
```suggestion
``CoqMakefile``. In particular, you can declare custom actions to
run before or after the build process. Similarly you can
```